### PR TITLE
UPGRADE eslint to v7

### PR DESCRIPTION
**Pull request by RecuencoJones**
New year, fancy upgrades

### CODE REVIEW

**Review by moelders — COMMENTED** [review]
I undertand that disabling the new rules we keep backwards compatibility, but, the question is, as we have well defined rules, we should enable some of them. What do you think?

**Comment by RecuencoJones** [comment]
> I undertand that disabling the new rules we keep backwards compatibility, but, the question is, as we have well defined rules, we should enable some of them. What do you think?

I actually didn't disable all new rules, only disabled those that we usually don't care about / are too tightly coupled to each codebase!
We can however get together and review specific rules 😄

File: packages/eslint-config-adidas-es5/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,78 @@
+# 1.3.0
+
+- Updated ESLint to version 7.
+- Removed deprecated rules that were moved to `eslint-plugin-node`.
+- Added all missing rules for ES6, 7, 8... as `off`:
```
Comment (RecuencoJones) [inline packages/eslint-config-adidas-es5/CHANGELOG.md:5]: @moelders if you meant this, I've added to ES5 configuration all missing ES6+ rules just to be sure their defaults don't get applied incorrectly for ES5 codebases!

**Review by moelders — APPROVED** [review]
LGTM

**Pull request merged**